### ISSUE TITLE
Euro Truck Simulator 2 (227300)

### ISSUE THREAD

**Issue by kallestrop** [ValveSoftware/Proton#4730]
Since Proton update to version 6.3 my Euro Truck Simulator 2 triple screen setup (7680 x 1440) is broken. After start game recognize only single monitor layout and  menu screen of the game is located in the left side of the wine virtual desktop window instead of center. The rest of the window is filled with the blue background color of the wine desktop.

Tested on 2 Linux distributions (Debian 11 sid/bullseye and Linux Mint 20.1 Ulyssa and 3 desktop environments Cinnamon / Mate and XFCE

Issue occurs only on Cinnamon and Mate desktop environment. On XFCE all works fine like in previous Proton versions

Also Proton-6.5-GE-1 is NOT affected to this issue

**Comment by kisak-valve** [comment]
Hello @kallestrop, it's not clear if wine's virtual desktop setting is supported or not in Proton, but putting that aside for a moment, please clarify if you know if the game behaved as expected with an older mainline Proton version, and if it is, then what's the most recent known good version?

Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by kallestrop** [comment]
Hello,

thanks for the quick reply. 

- Wine's virtual desktop setting worked on all previous Proton versions (5.0 - 5.13 - even experimental before yesterdays update)


My system information:

https://gist.github.com/kallestrop/58ca1f73e1961232637c2a10099fb7e3


I attached 2 logs - one from working Proton 5.13-6 and one with not working Proton 6.3

[steam-227300_Proton_5_13-6_working.log](https://github.com/ValveSoftware/Proton/files/6250893/steam-227300_Proton_5_13-6_working.log)
[steam-227300_Proton_6-3_not-working.log](https://github.com/ValveSoftware/Proton/files/6250916/steam-227300_Proton_6-3_not-working.log)

**Comment by kisak-valve** [comment]
Thanks, this looks like some lines of interest from the log:
```
info:  Restoring display mode: 7680x1440@60
info:  Setting display mode: 7680x1440@60
5148.781:011c:0120:err:system:ChangeDisplaySettingsExW Changing L"\\\\.\\DISPLAY1" display settings returned -2.
5148.782:011c:0120:err:system:ChangeDisplaySettingsExW Changing L"\\\\.\\DISPLAY1" display settings returned -2.
warn:  DXGI: LeaveFullscreenMode: Failed to restore display mode
```

**Comment by kallestrop** [comment]
Maybe this is also important: I always have to set the left monitor as primary screen with 

`xrandr --output DisplayPort-0 --primary`
 
otherwise the output of the game was not correct because the left side of the left monitor is the origin.

**Comment by kattjevfel** [comment]
Seems the same as #4673 to me

**Comment by kallestrop** [comment]
I can confirm,  that my problem with broken wine virtual desktop screen layout for Euro Truck Simulator 2 is solved after release of Proton 6.3-2.
